Crash when cache URL isn't present

```
./dist/stayrtr-0.1-62-gcc539c4-linux-x86_64 -metrics.addr "127.0.0.1:9847" -cache http://127.0.0.1:8081/output.json
ERRO[0000] Error updating: HTTP 503 Service Unavailable 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7f7c4e]

goroutine 1 [running]:
main.(*state).updateFromNewState(0xc0000c6000)
        ./stayrtr.go:258 +0x4e
main.run()
        ./stayrtr.go:595 +0xaf5
main.main()
        ./stayrtr.go:488 +0x19
```

--- a/cmd/stayrtr/stayrtr.go
+++ b/cmd/stayrtr/stayrtr.go
@@ -379,6 +379,10 @@ func (s *state) routineUpdate(file string, interval int, slurmFile string) {
 	signal.Notify(signals, syscall.SIGHUP)
 	for {
 		delay := time.NewTimer(time.Duration(interval) * time.Second)
+		if (s.lastchange.IsZero()) {
+			log.Warn("Initial sync not complete. Refreshing every 30 seconds")
+			delay = time.NewTimer(time.Duration(30) * time.Second)
+		}
 		select {
 		case <-delay.C:
 		case <-signals:

